.run/go build github.com_octopusdeploy_octopus-permissions-controller_cmd.run.xml

```diff
@@ -2,7 +2,7 @@
   <configuration default="false" name="go build github.com/octopusdeploy/octopus-permissions-controller/cmd" type="GoApplicationRunConfiguration" factoryName="Go Application" nameIsGenerated="true">
     <module name="octopus-permissions-controller" />
     <working_directory value="$PROJECT_DIR$" />
-    <parameters value="--webhook-cert-path ./certificates" />
+    <parameters value="--webhook-cert-path ./certificates --metrics-bind-address=&quot;:8443&quot; --metrics-secure=false" />
```

Comment:
We may want to consider updating this to bind to a different port, as this would frequently be used for gRPC if running in tandem with a dev octopus server